wx/lib/pdfviewer/bezier.py

```diff
@@ -58,7 +58,7 @@ def compute_points(controlpoints, nsteps=30):
     """
     Input 4 control points as :class:`RealPoint` and convert to vec2d instances.
     compute the nsteps points on the resulting curve and return them
-    as a list of :class:`wx.Point`
+    as a list of :class:`Point`
```

Comment:
The intra-doc links like this should keep the fully qualified names, otherwise there can be ambiguous names that Sphinx can't figure out and it just makes a (usually incorrect) guess.  I made a pass over everything a while back fixing those items that Sphinx gave warnings about, so maybe you just inadvertently overwrote my changes by starting with an older version of the files.  

While you are fixing these then it would be good to also add the leading `wx.` to the other links, for consistency.